src/ui/navbar-header.svelte

```diff
@@ -66,5 +105,20 @@
     .rcb-header-layout {
       width: auto;
     }
+
+    .rcb-back {
+      display: block;
+    }
+
+    .rcb-close {
+      display: none;
+    }
+
+    .rcb-header-multiline-layout {
+      all: unset;
+      display: flex;
+      flex-direction: column;
```

Comment:
these three are already defined with the same value above

Reply:
Removed